ceph.spec.in

```diff
@@ -839,7 +839,7 @@ rm -rf %{buildroot}
 %{_sbindir}/ceph-create-keys
 %{_sbindir}/rcceph
 %dir %{_libexecdir}/ceph
-%{_libexecdir}/ceph/ceph_common.sh
+%{_libdir}/ceph/ceph_common.sh
```

Comment:
libraries go to `/usr/lib`[1]. so i don't think we should put the (architecture independent) executables there. a more appropriate place is `/usr/libexec`, where binaries run by programs are located, because this script is an executable not supposed to be launched by user. and more specifically, fedora packaging policy puts:

> Libexecdir (aka, /usr/libexec on Fedora systems) should only be used as the directory for executable programs that are designed primarily to be run by other programs rather than by users.

see https://fedoraproject.org/wiki/Packaging:Guidelines#Libexecdir . so this packaging complies to the fedora packaging guidelines.

but debian policy states that 

>  In particular, a subdirectory of /usr/lib may be used by a package (or a collection of packages) to hold a mixture of architecture-independent and architecture-dependent files. However, when a directory is entirely composed of architecture-independent files, it should be located in /usr/share.

so `-DCMAKE_INSTALL_LIBEXECDIR=/usr/lib` in `debian/rules` complies to this policy, and is not a workaround.

---
[1] https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#usrlibLibrariesForProgrammingAndPa
[2] https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#usrlibexec